cluster-scope/overlays/prod/emea/jerry/groups/cluster-admins.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: cluster-admins
+users:
+  - durandom
+  - humairak
```

Comment:
can you remove lower case (this is a left over from migration, we need to at some point do this for all groups)